h/services/subscription.py

```diff
@@ -0,0 +1,82 @@
+from sqlalchemy import func
+from sqlalchemy.orm import Session
+from webob.cookies import SignedSerializer
+
+from h.models import Subscriptions
+from h.security import derive_key
+
+
+class InvalidUnsubscribeToken(Exception):
+    ...
+
+
+class SubscriptionService:
+    # Would be nice if this was something to do with the DB model
+    class Type:
```

Comment:
Yeah it's a bit too much work for this when we only have one type.

The DB model already has a type so completely ignoring it feels wrong.